lib/reporter.js

```diff
@@ -66,12 +67,21 @@ export default class Reporter {
     })
   }
 
-  normalizePath (path) {
-    return path.replace('file:///', '')                         // Randomly inserted file url protocols
-               .replace(/[/]/g, '\\')                           // Temp switch for Windows home matching
-               .replace(fs.getHomeDirectory(), '~')             // Remove users home dir for apm-dev'ed packages
-               .replace(/\\/g, '/')                             // Switch \ back to / for everyone
-               .replace(/.*(\/(app\.asar|packages\/).*)/, '$1') // Remove everything before app.asar or pacakges
+  normalizePath (pathToNormalize) {
+    const absolutePath = pathToNormalize.replace('file:///', '')
+    let relativePath
+    if (this.isBundledFile(absolutePath)) {
+      relativePath = path.relative(this.resourcePath, absolutePath)
+    } else {
+      const homeDirPath = fs.getHomeDirectory()
+      relativePath = absolutePath
+        .replace(/[/]/g, '\\')                           // Temp switch for Windows home matching
+        .replace(homeDirPath.replace(/[/]/g, '\\'), '~') // Remove users home dir
+        .replace(/.*(\/(app\.asar|packages\/).*)/, '$1') // Remove everything before app.asar or pacakges
```

Comment:
pacakges should be packages - might as well correct that - was wrong previously too.